src/main/java/org/opensearch/security/filter/SecurityRestFilter.java

```diff
@@ -138,19 +145,53 @@ private boolean userIsSuperAdmin(User user, AdminDNs adminDNs) {
         return user != null && adminDNs.isAdmin(user);
     }
 
+    private boolean authorizeRequest(RestHandler original, RestRequest request, RestChannel channel, User user) throws Exception {
+        if (original instanceof RestSendToExtensionAction) {
+            List<RestHandler.Route> outOfProcessRoutes = original.routes();
```

Comment:
Why `outOfProcessRoutes`? The routes may be declared by an extension using the SDK, but the routes are present in OpenSearch and opensearch will forward the request to the extension that has registered the route.

Reply:
it failed code-ql due to extensions keyword here, so I changed the name, but I'll change it back now and suppress the check for this file.

Reply:
What about `routes`?

Reply:
we need to suppress it for the file otherwise it fails for `RestSendToExtensionAction`